### PR TITLE
Website

### LINKED ISSUE #21: Wordpress is not Communicating with MySQL
Once both containers are started, Wordpress will not connect with MySQL. 

Steps to Reproduce: 

1. Wordpress will show an Install Page at the beginning, go in and delete /var/www/html/wp-config.php and reload Wordpress.
2. It will then display https://codex.wordpress.org/File:install-step3.png, enter in the required information
3. Hit Continue

### PR DESCRIPTION
**Pull request by pixel5919**
Potentially fixes #21.

### CODE REVIEW

**Comment by iankronquist** [comment]
Looks good to me!

File: docker-compose.yml [hunk 1]
```diff
@@ -10,6 +10,6 @@ mysql:
 wordpress:
   build: .
   ports:
-    - "8000:80"
+    - "80:80"
```
Comment (iankronquist) [inline docker-compose.yml]: Please update line 25 in the Vagrantfile to forward port 80 on the VM guest to port 8000 on the host.

**Comment by pixel5919** [comment]
On fresh new Vagrant after installing everything on the website: `Cannot connect to the Docker daemon. Is the docker daemon running on this host?`

**Comment by iankronquist** [comment]
Are you in the docker group after provisioning? Do you need to log out and log back in again?

**Comment by pixel5919** [comment]
Yes, logging out and back in allowed the honey pot to work

**Pull request merged**